plugins/wp-graphql-headless-webhooks/src/Handlers/WebhookHandler.php

```diff
@@ -0,0 +1,48 @@
+<?php
+namespace WPGraphQL\Webhooks\Handlers;
+
+use WPGraphQL\Webhooks\Entity\Webhook;
+use WPGraphQL\Webhooks\Handlers\Interfaces\Handler;
+
+/**
+ * Class WebhookHandler
+ *
+ * Sends the webhook to the configured URL when an event is triggered.
+ */
+class WebhookHandler implements Handler {
```

Comment:
We support one type of handler which is an HTTP Request handler that sends a POST or GET request to an endpoint